packages/geo/package.json

```diff
@@ -52,8 +74,8 @@
     "@accelint/constellation-tracker": "workspace:*",
     "@accelint/typescript-config": "workspace:*",
     "@accelint/vitest-config": "workspace:*",
-    "@types/geodesy": "^2.2.8",
```

Comment:
the types dependency is removed, but the actual dependency is still included. is this intentional?